client/slack/invite_form.js

```diff
@@ -1,27 +1,100 @@
+import React, { Component } from 'react';
+import { Form, Message } from 'semantic-ui-react';
 import $ from 'jquery';
 
-$(() => {
-    $('#slack_invite_form')
-        .form({
-            fields: {
-                email: {
-                    identifier: 'email',
-                    rules: [
-                        {
-                            type: 'email',
-                            prompt: 'Please enter a valid email'
-                        }
-                    ]
+import { retrieveTaskResult } from '../lib/task';
+
+
+class SlackInviteForm extends Component {
+    state = {
+        email: '',
+        acceptTos: false,
```

Comment:
This looks like `accept Tos` instead of `accept TOS`. May want to capitalize `TOS`